checkov/terraform/checks/resource/gcp/GoogleCloudSqlBackupConfiguration.py

```diff
@@ -1,17 +1,28 @@
-from checkov.common.models.enums import CheckCategories
-from checkov.terraform.checks.resource.base_resource_value_check import BaseResourceValueCheck
+from checkov.common.models.enums import CheckCategories, CheckResult
+from checkov.terraform.checks.resource.base_resource_value_check import BaseResourceCheck
 
 
-class GoogleCloudSqlBackupConfiguration(BaseResourceValueCheck):
+class GoogleCloudSqlBackupConfiguration(BaseResourceCheck):
     def __init__(self):
         name = "Ensure all Cloud SQL database instance have backup configuration enabled"
         id = "CKV_GCP_14"
         supported_resources = ['google_sql_database_instance']
         categories = [CheckCategories.NETWORKING]
         super().__init__(name=name, id=id, categories=categories, supported_resources=supported_resources)
 
-    def get_inspected_key(self):
-        return 'settings/[0]/backup_configuration/[0]/enabled'
+    def scan_resource_conf(self, conf):
+        if 'master_instance_name' in conf:
+           return CheckResult.PASSED
```

Comment:
you can just leave the subclass as it is and adjust the check like this
```python
    def scan_resource_conf(self, conf):
        if 'master_instance_name' in conf:
            return CheckResult.UNKNOWN
        return super().scan_resource_conf(conf)

    def get_inspected_key(self):
        return "settings/[0]/backup_configuration/[0]/enabled"
```
similar like here
https://github.com/bridgecrewio/checkov/blob/master/checkov/terraform/checks/resource/aws/ALBDropHttpHeaders.py

Reply:
Way cleaner! Thanks for the sugggestion.